lib/srv/desktop/windows_server.go

```diff
@@ -1324,6 +1325,8 @@ type generateCredentialsRequest struct {
 	username string
 	// domain is the Windows domain
 	domain string
+	// ad is true if we're connection to AD-joined desktop
+	ad bool
```

Comment:
Do we need an extra field, or can we tell based on `domain` being empty?

Reply:
If we have domain configured for WDS it will be present for both AD and non-AD hosts (at least in case of static hosts from config), so we'd either have to change that or add new field. New field is cleaner in my opinion

Reply:
`desktop.GetDomain()` would return the empty string, even if WDS is configured for AD, right? (Because if you set `ad: false` in `static_hosts` then we won't set the domain.

Doesn't really matter though.